### DISCUSSION TITLE
Type hint for `pytestconfig` fixture

### DISCUSSION THREAD

**Discussion by ethiy** [pytest-dev/pytest#9578] [Ideas] [answered]
Right now to annotate the `pytestconfig` fixture one has to import the `_pytest.config.Config` object.
However, the underscore present in the module name denotes that this class is not supposed to be reachable from outside the package.
My question would be: can this class be exposed in the `pytest` `__init__.py` file, or it is not meant to be at all? In that case, what would the solution be in order to properly annotate this fixture.

**Comment by jwodder** [comment, marked as answer]
There is already work underway to expose classes needed for type annotations in #7469, and this seems to be scheduled for pytest 7.  Not sure what to do until that's released, though.
